### PR TITLE
Removes overview dark theme(#578)

### PR DESCRIPTION
**Pull request by fragm3**
Fixes issue #[578]

Dark background wont be there anymore. Overview has only light theme, irrespective of theme set by user.

### CODE REVIEW

**Comment by rishiraj824** [comment]
@fragm3 Please fix travis errors. Thanks!

File: src/components/Overview/Overview.react.js [hunk 1]
```diff
@@ -149,8 +148,7 @@ class Overview extends Component {
       }
     const actions = <RaisedButton
       label="Cancel"
-      backgroundColor={
-        UserPreferencesStore.getTheme()==='light' ? '#4285f4' : '#19314B'}
+      backgroundColor='#607D8B'
```
Comment (uday96) [inline src/components/Overview/Overview.react.js]: @fragm3 Shouldn't the color be `#4285f4` ?

Comment (fragm3) [inline src/components/Overview/Overview.react.js]: No, #607D8B is working fine. I think earlier code was not right. I have testes it, its working

Comment (isuruAb) [inline src/components/Overview/Overview.react.js]: @fragm3  #4285f4 this one is the new colour.#607D8B is older one .please swap that.

**Comment by isuruAb** [comment]
@fragm3  please send your PRs according to the Pull request template. you should provide preview link with your PRs. 
It should be Fixes issue #578 not Fixes issue #[578]
👍

**Comment by mariobehling** [comment]
@fragm3 Thanks for your contribution. Please try to integrate your work with best practices more step by step to make it easier for everyone to collaborate. Cheers. :-)

**Pull request merged**